pages/_app.tsx

```diff
@@ -21,12 +29,28 @@ function MyApp({
     }
   }, [])
   return (
-    <ThemeProvider theme={theme}>
-      <StylesProvider injectFirst>
-        <CssBaseline />
-        <Component {...pageProps} err={err} />
-      </StylesProvider>
-    </ThemeProvider>
+    <div>
+      <ThemeProvider theme={theme}>
+        <StylesProvider injectFirst>
+          <CssBaseline />
+          <Component {...pageProps} err={err} />
+        </StylesProvider>
+      </ThemeProvider>
+      <Head>
+        {/* Import CSS for nprogress */}
+        <link rel="stylesheet" type="text/css" href="/nprogress.css" />
+      </Head>
```

Comment:
The head element is below the page. How will the loader show above the page

Reply:
sorry i fix this